cg/cli/deliver/base.py

```diff
@@ -89,9 +87,10 @@ def deliver_case(
     if not case:
         LOG.error(f"Could not find case with id {case_id}")
         return
+    pseudo_workflow: str = get_pseudo_workflow(case=cases[0])
```

Comment:
this is not needed with my suggestion

With the current implementation this would also be needed in the upload api

Reply:
implemented the changes